packages/vaex-ml/setup.py

```diff
@@ -21,9 +21,11 @@
       author=author,
       author_email=author_email,
       install_requires=install_requires_ml,
+      extras_require={'ml.tensorflow': ['tensorflow>=2.1.0', 'tensorflow-io>=0.12.0']},
```

Comment:
This is not easy to type, let's call it all

Reply:
Done